crates/ruff_python_formatter/src/verbatim.rs

```diff
@@ -573,33 +596,275 @@ impl Format<PyFormatContext<'_>> for TrailingFormatOffComment<'_> {
     }
 }
 
-struct VerbatimText(TextRange);
+/// Stores the indentation of a statement by storing the number of indentation characters.
+/// Storing the number of indentation characters is sufficient because:
+/// * Two indentations are equal if they result in the same column, regardless of the used tab size.
+///   This implementation makes use of this fact and assumes a tab size of 1.
+/// * The source document is correctly indented because it is valid Python code (or the formatter would have failed parsing the code).
+#[derive(Copy, Clone)]
+struct Indentation(u32);
```

Comment:
There's already an `Indentation` in ruff_python_parser, do we want to give this one a different name?

Reply:
Do you have a recommendation? I wasn't able to come up with a good name. 

I'm not too concerned about the naming conflict, considering that this is a private type.